docs/architecture-decisions.md

```diff
@@ -0,0 +1,11 @@
+# Architecture decisions
+
+## Deployment models
+Kyma-operator (and component operators) can run in following modes:
+- in-cluster - regular deployment in the kubernetes cluster where kyma should be deployed
+- control-plane - deployment on central kubernetes cluster that manages multiple kyma installations (installing kyma on the remote clusters)
```

Comment:
For control-plane, we identified 2 possibilities which are not excluding each other, see: https://github.com/kyma-project/community/pull/654/files#diff-9a15a3ecf03a4115da73886331f172eefc9e2d8c4a23b1b4f15dfe7f38396493R89-R95

vs https://github.com/kyma-project/community/pull/654/files#diff-9a15a3ecf03a4115da73886331f172eefc9e2d8c4a23b1b4f15dfe7f38396493R97-R107

Reply:
We will extend it when PoC is validated by performance tests.